lib/hardware/modules/div/iob_div_subshift_frac/iob_div_subshift_frac.py

```diff
@@ -13,22 +22,196 @@ def setup(py_params_dict):
+                    {
+                        "name": "remainder",
+                        "width": "DATA_W",
+                        "direction": "output",
+                    },
+                ],
+            },
         ],
-        "blocks": [
+        "wires": [
             {
-                "core_name": "iob_reg",
-                "instance_name": "iob_reg_inst",
+                "name": "divisor_reg",
+                "descr": "divisor_reg wire",
+                "signals": [
+                    {"name": "divisor_reg", "width": "DATA_W"},
+                ],
+            },
+            {
+                "name": "quotient_int",
+                "descr": "quotient_int wire",
+                "signals": [
+                    {"name": "quotient_int", "width": "DATA_W"},
+                ],
+            },
+            {
+                "name": "incr",
+                "descr": "incr wire",
+                "signals": [
+                    {"name": "incr", "width": 1},
+                ],
             },
             {
-                "core_name": "iob_reg_e",
-                "instance_name": "iob_reg_e_inst",
+                "name": "res_acc",
+                "descr": "res_acc wire",
+                "signals": [
+                    {"name": "res_acc", "width": "DATA_W+1"},
+                ],
+            },
+            {
+                "name": "pc",
+                "descr": "pc wire",
+                "signals": [
+                    {"name": "pc", "width": 2},
+                ],
+            },
+            {
+                "name": "div_frac",
+                "descr": "Division interface",
+                "signals": [
+                    {
+                        "name": "dividend",
+                    },
+                    {
+                        "name": "divisor",
+                    },
+                    {
+                        "name": "quotient_int",
+                    },
+                    {
+                        "name": "remainder",
+                    },
+                ],
+            },
+        ],
+        "blocks": [
+            {
+                "core_name": "iob_reg",
+                "instance_name": "divisor_reg0",
+                "parameters": {
+                    "DATA_W": "DATA_W",
+                    "RST_VAL": "1'b0",
+                },
+                "connect": {
+                    "clk_en_rst": "clk_en_rst",
+                    "data_i": "divisor_i",
+                    "data_o": "divisor_reg",
+                },
             },
             {
                 "core_name": "iob_div_subshift",
-                "instance_name": "iob_div_subshift_inst",
+                "instance_name": "div_subshift0",
+                "parameters": {
+                    "DATA_W": "DATA_W",
+                },
+                "connect": {
+                    "clk_en_rst": "clk_en_rst",
+                    "div": "div_frac",
+                },
```

Comment:
The `status` port of the [iob_div_subshift](https://github.com/IObundle/iob-soc/pull/911/files#diff-356cb09132d29cd39cfa994ac65b1082cd1e3c51814ce295926e8684b594a434R26) module does not seem to be connected.